21_03_10_server/src/de/telran/LoadSender.java

```diff
@@ -0,0 +1,47 @@
+package de.telran;
+
+import java.io.IOException;
+import java.net.*;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class LoadSender implements Runnable {
+
+    private static final String UDP_SERVER_HOST = "localhost";
+
+    private final int udpServerPort;//порт LoadReceiver
+    private final AtomicInteger tcpConnectionsCounter;
+    private int sendingFrequency;
+
+    public LoadSender(int udpServerPort, AtomicInteger tcpConnectionsCounter, int sendingFrequency) {
+        this.udpServerPort = udpServerPort;
+        this.tcpConnectionsCounter = tcpConnectionsCounter;
+        this.sendingFrequency = sendingFrequency;
+    }
+
+    public void run() {
+
+        try {
+            InetAddress inetAddress = InetAddress.getByName(UDP_SERVER_HOST);
+            DatagramSocket udpSocket = new DatagramSocket(udpServerPort);
+
+            String line = String.valueOf(tcpConnectionsCounter);
```

Comment:
you should get the value of tcpConnectionsCounter first. But maybe I have a mistake.